services/time/Cargo.toml

```diff
@@ -28,6 +28,7 @@ protobuf = "2.8.0"
 
 [dev-dependencies]
 exonum-testkit = { version = "0.12.0", path = "../../testkit" }
+exonum-cli = { version = "0.12.0", path = "../../components/cli" }
```

Comment:
I think that components should contains only crates from which exonum depends.

Reply:
exonum-build is not the dependency of the exonum

Reply:
@vitvakatu you meant `exonum-cli` ?

Reply:
No, I meant exonum-build which is also placed inside `components` though it's not the component of the Core

Reply:
Moved the crate outside of components directory after private conversation